### PR TITLE
simplify login container

### PR DESCRIPTION
**Pull request by PerfectPixel**
I would like to simplify the login container in the following way:

1. separate imports: first third-party node_modules, then local imports
2. use single quotes (or double quotes, as long as it is consistent)
3. split up large render functions into smaller functions to avoid large conditional blocks
4. move prop-types outside the class definition
5. use default props instead of extensive ternary operators within the render function

### CODE REVIEW

File: src/login/LoginContainer.js [hunk 1]
```diff
@@ -33,50 +20,72 @@ export default class LoginContainer extends Component {
...
         );
     }
 }
+
+LoginContainer.defaultProps = {
+    containerClass: 'login__container',
+    formClass: 'login__form',
+    formInputClass: 'login__input',
+    formInputPlaceholder: 'Your email-address',
+    formSubmitButtonClass: 'login__submit'
```
Comment (tarcon) [inline src/login/LoginContainer.js]: Cool. I didn't know about default props.

File: src/login/LoginContainer.js [hunk 2]
```diff
@@ -70,11 +70,11 @@ export default class LoginContainer extends Component {
...
-    containerClass: 'login__container',
-    formClass: 'login__form',
-    formInputClass: 'login__input',
-    formInputPlaceholder: 'Your email-address',
-    formSubmitButtonClass: 'login__submit'
+    containerClass:         'login__container',
+    formClass:              'login__form',
+    formInputClass:         'login__input',
+    formInputPlaceholder:   'Your email-address',
+    formSubmitButtonClass:  'login__submit'
```
Comment (tarcon) [inline src/login/LoginContainer.js]: Is there any way of telling Intellij autoformat to respect your effort? :D

File: src/login/LoginContainer.js [hunk 3]
```diff
@@ -33,50 +20,72 @@ export default class LoginContainer extends Component {
...
+    alertMessageClass:      PropTypes.string,
+    emailRequestedCallback: PropTypes.func,
+    formClass:              PropTypes.string,
+    formInputClass:         PropTypes.string,
+    formInputPlaceholder:   PropTypes.string,
+    formSubmitButtonClass:  PropTypes.string,
+    headlineRender:         PropTypes.element,
+    loginContainerClass:    PropTypes.string,
+    subHeadlineRender:      PropTypes.element
+};
```
Comment (ulich) [inline src/login/LoginContainer.js]: was there a reason why you moved propTypes and defaultProps down here? I like to define them in the class at the top, so you immediately see the "API" of the component: https://github.com/d-i-p/customer-portal-ui/blob/master/src/claim/ClaimForm.js and you if you want to rename the class, you just have to rename once thing and not several ones

File: src/login/LoginContainer.js (hunk 2 above)
Comment (PerfectPixel) [reply src/login/LoginContainer.js]: I don't know and I do not care too much.

File: src/login/LoginContainer.js (hunk 3 above)
Comment (PerfectPixel) [reply src/login/LoginContainer.js]: No specific reason. According to the docs the definition as static properties requires additional dependencies. Each component uses them differently, we can inline them as well.

Renaming will never be easy as you have tests and stories etc in different files along with other im- and exports.

**Pull request merged**